src/Analysis.R

```diff
@@ -42,7 +32,8 @@ penguins_data <- penguins_data |>
 ## Assign labels:
 # (see https://allisonhorst.github.io/palmerpenguins/reference/penguins.html
 #   for more info):
-penguins_data <- penguins_data |> set_variable_labels(
+penguins_data <- penguins_data |>
+  set_variable_labels(
```

Comment:
This is just a matter of style... 🙂 
(both are accepted in the [tidyverse style guide](https://style.tidyverse.org/pipes.html#short-pipes))